denis/configure.py

```diff
@@ -2,6 +2,8 @@
 
 from datetime import datetime
 from argparse import ArgumentParser as ap
+from pathlib import Path
+import os
 
```

Comment:
I think it might be cleaner to have the start.py touch a `/tmp/reload` file every time it reloads and then just compare the mtime from `stat`ing the db vs that file so that you don't need to litter the calls to `dirty()` throughout the whole file relying on the fact that if an update occurs, the db will be written to and have its mtime bumped.

Reply:
Splitting this commit into https://github.com/underground-software/singularity/pull/277